spec/features/api/v1/murals_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+feature 'murals api' do
+  before(:each) do
+    User.create(username: "super_admin", password: "password", email: "admin@gmail.com", admin: true)
+    @user = User.create(username: "notanadmin", password: "password1234", email: "notanadmin@gmail.com")
+    @mural = Mural.create(name: 'Big Fish', location: 'Martin Luther King Dr', user_id: @user.id, description: "This is a description", rating: '4', photo_url: 'http://www.petsmart.com/learning-center/fish-care/the-right-food-to-feed-your-fish/A0009.html')
+    Review.create(rating: 2, review: "This is a bad review", user_id: @user.id, mural_id: @mural.id)
+  end
+
+  scenario 'User accesses api info' do
+    visit api_v1_murals_path
+
+    expect(page).to have_content "id"
+    expect(page).to have_content "name"
+    expect(page).to have_content "location"
+  end
+
+  scenario 'User accesses individual mural info' do
```

Comment:
A few additional expect statements would be good here, just to check you have the right response code from the fetch call, and the correct body.  That way, if something interferes with your fetch, you have a more accurate system to track the problem.